pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md

```diff
@@ -0,0 +1,225 @@
+---
```

Comment:
The filename has a misspelling - it think you wanted 'eks-cluster-iam-policies-and-roles', not polices.  :-) 
These show up in the URI's for the page so we need to fix this...